MauroDataMapper-Plugins/mdm-plugin-explorer
# gh-37: Add Data Specification PDF exporter

Closes #37 

To test this work you will need to:
1. Create a Data Specification in the MDE
2. Log out of the MDE and log into Mauro UI as admin
3. Go to the Data Specification you created in step 2
4. Click on the export icon
5. Choose "PDF Data Specification Exporter"
6. Export

Note that this is the first iteration of the PDF. We will come back and export more detailed information, but the information contained within will be enough to get the system up and running for demo purposes. Also note that some of the formatting in the PDF could be improved, but for now it's just about getting the info out there.

Tests have been written to prove this works.
